README.md

```diff
@@ -1,32 +1,18 @@
-# HACBS Release bundles
+# Release Service Catalog
 
-Release service bundles is a curated Tekton bundles catalog that contains bundles that are used by the
-[Appstudio Release operator](https://github.com/redhat-appstudio/release-service) to process releases in the context of
+This repository contains a collection of Tekton resources to be used with the Release Service. The main purpose of these
```

Comment:
A link to the release Service like there was would be nice

Reply:
Done.